docs/source/tools/canton-console.rst

```diff
@@ -0,0 +1,62 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Canton Console
+##############
+
+Introduction
+============
+
+Canton offers a console where administrative or debugging commands can be run.
+
+When you run the Sandbox using ``daml start`` or ``daml sandbox``, you are effectively starting an
+in-memory instance of Canton with a single domain and a single participant.
+
+As such, you can interact with the running Sandbox using the Canton Console, just like you would
+in a production environment.
+
+The purpose of this page is to give a few pointers on how the Canton Console can be used to
+interact with a running Sandbox. For an in-depth guide on how to use this tool against a production,
+staging or testing environment, consult the main documentation for the Canton Console.
+
+Run the Canton Console against the Sandbox
+==========================================
+
+Once you have a Sandbox running locally (for example after running ``daml start`` or ``daml sandbox``)
+you can start the Canton Console with the following command (in a separate terminal from the one where
```

Comment:
```suggestion
you can start the console with the following command (in a separate terminal from the one where
```

Reply:
Addressed in https://github.com/digital-asset/daml/pull/15292/commits/f2f3f3bfb8c5e4ebc0c8c539606e52f83b98d132